python/tank/bootstrap/bundle_downloader.py

```diff
@@ -0,0 +1,50 @@
+import os
+from sgtk import hook
+from sgtk import LogManager
+
+log = LogManager.get_logger(__name__)
+
+
+# FIXME: This class has a terrible name, we need to rename it.
```

Comment:
if you are not happy with the nqame, please think of a better one before merging :)